### ISSUE TITLE
Tony moving fine but not firing the laser

### ISSUE THREAD

**Issue by huffeec** [DoESLiverpool/somebody-should#1774]
User today reported that the laser isn't firing. 
I couldn't see anything immediately obvious wrong. Although the laser psu light is off, I can't remember if it's usually lit?

![20230222_135954](https://user-images.githubusercontent.com/4748179/220650867-aa24a562-2b8b-4020-bf00-1a6291cb163e.jpg)

@DoESLiverpool/laser-maintenance

**Comment by RussCoty** [comment]
Hi, Brian (Subbuteo) reported this to me on Monday night. Then I came down with THE VIRUS. He had written on the board. 

I had intended to raise this issue. Apologies.

**Comment by Sean-anotherone** [comment]
**Provisional** triage completed.

The MCB for the HV Laser PSU had tripped, reset it to no effect.

Laser HV PSU is confirmed as receiving 240vac, but the power LED is not lit.  

Suspect HV PSU is dead, and took the breaker out when it failed, resetting the breaker does not result in a trip because whatever blew up is now open circuit.

**Another competent person should verify this diagnosis before a new supply is committed.**
I would suggest another supply like the new one in Gerald would be the way to go, once the fault is confirmed.

**Comment by Sean-anotherone** [comment]
Also, I can't remember if I wired up that LED or not, and forgot to look before putting the panels back on.

**Comment by magman2112** [comment]
Re-creating this comment as it apparently didn’t get stored last time.

After duplicating the testing that @Sean-anotherone completed above, I can confirm that the laser power supply has failed and needs replacement.

To give us a degree of compatibility with the Laser PSU in Gerald, I propose that we install the same PSU in Gerald. Here is a link to this PSU on Amazon.

https://www.amazon.co.uk/Cloudray-80-100W-Supply-Engraver-MYJG-100/dp/B07C2QN5ZD/ref=sr_1_4_sspa?crid=2JRFWMX0TWPJF&keywords=laser+power+supply&qid=1641238129

**Comment by amcewen** [comment]
I've ordered one of those.  Estimated delivery is March 31 - April 3.

**Comment by amcewen** [comment]
The power supply has arrived.  I've left it on Tony's bed.

**Comment by magman2112** [comment]
The new laser tube power supply has been installed and is working.

As the new power supply has a power meter capability, @Sean-anotherone ans I found that the laser tube was firing at 27mA. Checking the specifications for the tube, we found that the recommended maximum current is 22mA. We have updated the configuration file for the laser controller to adjust this setting back to the recommendation level, as this will considerably increase the life of the tube. Subsequent tests showed that cuts are hardly changed from the previous settings, though this will need to be verified in the near future.

I’ll leave this issue open for a short while, to get feedback from other users.
